app/javascript/packs/components/_button.scss

```diff
@@ -0,0 +1,17 @@
+input[type=submit]{
+    @apply h-10 flex-shrink-0 bg-gray-900 text-sm w-full text-white py-1 px-2 rounded;
+    &:hover{
+        @apply bg-gray-600;
+    }
+    @screen md{
+        @apply h-14;
+    }
+}
+.btn{
+    &__primary{
+        @apply bg-gray-900 flex-shrink-0 h-12 text-white px-5 rounded flex items-center justify-center mx-1;
+        &:hover{
+            @apply bg-gray-600;
+        }
+    }
+}
```

Comment:
check indentation

Reply:
The indentation spaces were modified in this file